x-pack/plugins/security_solution/server/ui_settings.ts

```diff
@@ -216,6 +217,23 @@ export const initUiSettings = (
       requiresPageReload: true,
       schema: schema.boolean(),
     },
+    [ENABLE_VISUALIZATIONS_IN_FLYOUT_SETTING]: {
+      name: i18n.translate('xpack.securitySolution.uiSettings.enableVisualizationsInFlyoutLabel', {
+        defaultMessage: 'Enable visualizations in flyout',
```

Comment:
I fel like the word `visualizations` here is not descriptive enough. While you and I understand what it means and does, I don't think other people at Elastic would, and even less end users... Should we mention `Session view` and `Analyzer graph` directly here?

Reply:
wdyt about this?

![image](https://github.com/user-attachments/assets/b2d283e9-b776-4acb-951a-807f58b58915)

Reply:
Yup, let's merge this as is and get with @nastasha-solomon to see what she thinks